Source/Core/EllipsoidTangentPlane.js

```diff
@@ -230,5 +314,56 @@ define([
         return result;
     };
 
+    var scratchOffset = new Cartesian3();
+    var scratchScale = new Cartesian3();
+    /**
+     * Computes an OrientedBoundingBox given extents in the east-north-up space of the tangent plane.
+     *
+     * @param {Number} minX Minimum x extent in tangent plane space.
+     * @param {Number} maxX Maximum x extent in tangent plane space.
+     * @param {Number} minY Minimum y extent in tangent plane space.
+     * @param {Number} maxY Maximum y extent in tangent plane space.
+     * @param {Number} minZ Minimum z extent in tangent plane space.
+     * @param {Number} maxZ Maximum z extent in tangent plane space.
+     * @param {OrientedBoundingBox} [result] The object onto which to store the result.
+     * @returns {OrientedBoundingBox} The modified result parameter or a new OrientedBoundingBox instance if one was not provided.
+     */
+    EllipsoidTangentPlane.prototype.extentsToOrientedBoundingBox = function(minX, maxX, minY, maxY, minZ, maxZ, result) {
```

Comment:
I figured that would probably be the case, but it does require access to the private `_xAxis`/`_yAxis`. Should I instead add a way to get those? (a property